### PR TITLE
API for changing request state

### LINKED ISSUE #85: [BACKEND] Implement "change request status" API.

### PR DESCRIPTION
**Pull request by yililiu8**

### CODE REVIEW

File: backend/API_DOC.md [hunk 1]
```diff
@@ -195,6 +195,41 @@ Endpoint to create and errand.
...
+
+**Method** : `POST`
+
+**Auth Required** : YES
+
+**Body Parameters** :
+
+```json
+{
+  "id": "[int, required id of the item]",
```
Comment (git-adithyanair) [inline backend/API_DOC.md]: Maybe change the description to `id of request` to make it more clear. I though it was the item id lmao

File: backend/API_DOC.md [hunk 2]
```diff
@@ -195,6 +195,41 @@ Endpoint to create and errand.
...
+  "user_id": "[int, id of user who created the request]", 
+  "community_id": "[int, id of community that request belongs to]", 
+  "status": "[RequestStatus, the status of the request (pending, in_progress, completed)]", 
+  "errand_id": "[int, id of errand associated with request, could be null]", 
+  "store_id": "[int, id of store that request is associated with]", 
+
+}
+```
+
+**Extra notes** : Any extra notes here.
```
Comment (git-adithyanair) [inline backend/API_DOC.md]: Probably don't need this

**Comment by git-adithyanair** [comment]
Everything looks good

File: backend/API_DOC.md (hunk 1 above)
Comment (yililiu8) [reply backend/API_DOC.md]: ohh yes thanks for catching that, forgot to change it when i copy and pasted lmao

**Pull request merged**